docs/create-packages/Symbol-Packages-snupkg.md

```diff
@@ -33,20 +33,19 @@ nuget pack MyPackage.nuspec -Symbols -SymbolPackageFormat snupkg
 nuget pack MyPackage.csproj -Symbols -SymbolPackageFormat snupkg
 ```
 
-If you're using dotnet.exe or MSBuild, use the following steps to create a .snupkg file in addition to the .nupkg file:
+For dotnet.exe:
 
-1. Add the following properties to your .csproj file:
+```
+dotnet pack MyPackage.csproj -p:IncludeSymbols=true -p:SymbolPackageFormat=snupkg
```

Comment:
Add this as an additional/alternate way to generate snupkg, and not the only way to do it.